ProcessLib/Process.h

```diff
@@ -259,32 +260,34 @@ class Process
 	{
 		DBUG("Process output.");
 
-		std::string const property_name = "Result";
-
-		// Get or create a property vector for results.
-		boost::optional<MeshLib::PropertyVector<double>&> result;
-		if (_mesh.getProperties().hasPropertyVector(property_name))
-		{
-			result = _mesh.getProperties().template getPropertyVector<double>(
-			    property_name);
-		}
-		else
-		{
-			result =
-			    _mesh.getProperties().template createNewPropertyVector<double>(
-			        property_name, MeshLib::MeshItemType::Node);
+		// Copy result
 #ifdef USE_PETSC
-			result->resize(_x->getLocalSize() + _x->getGhostSize());
+		std::vector<double> x_copy(_x->getLocalSize() + _x->getGhostSize());
 #else
-			result->resize(_x->size());
+		std::vector<double> x_copy(_x->size());
 #endif
-		}
-
-		assert(result);
-
-		// Copy result
-		_x->copyValues(*result);
+		_x->copyValues(x_copy);
 
+		std::size_t const n_nodes = _mesh.getNNodes();
+		for (ProcessVariable& pv : _process_variables)
+		{
+			auto& output_data = pv.getOrCreateMeshProperty();
+
+			int const n_components = pv.getNComponents();
+			for (std::size_t node_id = 0; node_id < n_nodes; ++node_id)
+			{
+				MeshLib::Location const l(_mesh.getID(),
+				                          MeshLib::MeshItemType::Node, node_id);
+				for (int component_id = 0; component_id < n_components;
+				     ++component_id)
+				{
+					auto const global_index =
```

Comment:
Note to myself: Using global index is wrong here.
```
error: 0 | n 432, c 0, gi 432 -> 432
error: 0 | n 433, c 0, gi 1190 -> 433
```
Seems like when starting accessing ghost nodes the global index can jump, like 433 is ghost node and is way back in the dof table. (1190 is one of the ghost indices). But for other ranks it's slightly different:
```
error: 2 | 626 456 170 626
error: 2 | n 0, c 0, gi 875 -> 0
```
and
```
error: 1 | n 185, c 0, gi 618 -> 185
error: 1 | n 186, c 0, gi 619 -> 186
```